### PR TITLE
Add PackageType as a mandatory field for creating lambda using container image

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
Be able to deploy lambda application under QuickSync strategy
```

### CODE REVIEW

File: pkg/app/piped/cloudprovider/lambda/client.go [hunk 1]
```diff
@@ -74,10 +74,9 @@ func (c *client) Apply(ctx context.Context, fm FunctionManifest, role string) er
 		Code: &lambda.FunctionCode{
 			ImageUri: aws.String(fm.Spec.ImageURI),
 		},
+		PackageType:  aws.String("Image"),
```
Comment (khanhtc1202) [inline pkg/app/piped/cloudprovider/lambda/client.go:77]: I thought about other use-cases when our users may want to be able to use other kinds of PackageType ( Zip or S3 object ). But currently, we only support container image packaging, so let me hard code this PackageType value here as is, and update this later when we have a plan to support a wide range of package types.

Comment (nghialv) [reply pkg/app/piped/cloudprovider/lambda/client.go:77]: This is an internal change so I am fine about it.
👍

**Pull request merged**